graylog2-server/src/main/java/org/graylog2/dashboards/widgets/DashboardWidget.java

```diff
@@ -91,6 +95,15 @@ public int getCacheTime() {
         return cacheTime;
     }
 
+    public TimeRange getTimeRange() {
+        this.checkTimeRange();
+        return timeRange;
+    }
+
+    private void checkTimeRange() {
```

Comment:
FWIW, I don't think it's necessary to extract that single check into its own method.